src/plugins/objects/pickables/pickables.plugin.ts

```diff
@@ -29,11 +30,11 @@ export const action: objectInteractionActionHandler = (details) => {
             itemId = itemIds.cabbage;
             break;
     }
-    const pickedItem = cache.itemDefinitions.get(itemId);
+    const pickedItem = findItem(itemId);
     setTimeout(() => {
-        details.player.sendMessage(`You ${details.option} the ${details.objectDefinition.name.toLowerCase()} and receive ${prefix} ${pickedItem.name.toLowerCase()}.`);
+        details.player.sendMessage(`You ${details.option} the ${details.objectConfig.name.toLowerCase()} and receive ${prefix} ${pickedItem.name.toLowerCase()}.`);
         details.player.playSound(2581, 7);
-        if (details.objectDefinition.name !== 'Flax' || Math.floor(Math.random() * 10) === 1) {
+        if (details.objectConfig.name !== 'Flax' || Math.floor(Math.random() * 10) === 1) {
             details.player.instance.hideGameObjectTemporarily(details.object, 30);
         }
         details.player.giveItem(pickedItem.id);
```

Comment:
`pickedItem.id` should be changed to `pickedItem.gameId`

Reply:
Fixed in the latest commit, good catch!